app/src/main/java/com/example/food_mvp/data/source/remote/fetchjson/GetJsonFromUrl.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.food_mvp.data.source.remote.fetchjson
+
+import android.os.AsyncTask
+import com.example.food_mvp.data.source.remote.OnFetchDataJsonListener
+import org.json.JSONObject
+import java.lang.Exception
+
+class GetJsonFromUrl<T> constructor(private val listener: OnFetchDataJsonListener<T>,
+                                    private val keyEntity: String) : AsyncTask<String?, Void?, String?>() {
```

Comment:
Em xem lại conventions về thứ tự param nhé
```suggestion
class GetJsonFromUrl<T>(
   private val keyEntity: String,
    private val listener: OnFetchDataJsonListener<T>
 
    ) : AsyncTask<String?, Void?, String?>() {
```

Reply:
Dạ